datafusion/core/src/datasource/physical_plan/parquet/statistics.rs

```diff
@@ -901,6 +921,29 @@ macro_rules! get_data_page_statistics {
                     }
                     Ok(Arc::new(builder.finish()))
                 },
+                // TODO file upstream in Arrowrs --
```

Comment:
I filed upstream as https://github.com/apache/arrow-rs/issues/6164

```suggestion
                // https://github.com/apache/arrow-rs/issues/6164
```